helix-view/src/clipboard.rs

```diff
@@ -104,7 +104,14 @@ pub fn get_clipboard_provider() -> Box<dyn ClipboardProvider> {
     // variable?
 
     if env_var_is_set("WEZTERM_UNIX_SOCKET") && binary_exists("wezterm") {
-        Box::new(provider::FallbackProvider::new())
+        if env_var_is_set("TMUX") && binary_exists("tmux") {
```

Comment:
Can you refactor this so that the tmux check and provider definition is not duplicated (it already exists on line 146) ?